obm-autoset/applyobmsetting.py

```diff
@@ -0,0 +1,350 @@
+                                       )
+
+        if rest_action == "post":
+            result_data = requests.post(url_command,
+                                        data=json.dumps(rest_payload),
+                                        headers=rest_headers,
+                                        timeout=rest_timeout,
+                                        verify=sslverify
+                                        )
+
+        if rest_action == "patch":
+            result_data = requests.patch(url_command,
+                                         data=json.dumps(rest_payload),
+                                         headers=rest_headers,
+                                         timeout=rest_timeout,
+                                         verify=sslverify
+                                         )
+    except requests.exceptions.Timeout:
+        return {'json':'', 'text':'',
+                'status':0,
+                'headers':'',
+                'timeout':True}
+
+    try:
+        result_data.json()
+    except ValueError:
+
+        if VERBOSITY >= 9:
+            print "restful: TEXT =\n"
+            print result_data.text
+        if VERBOSITY >= 6:
+            print "restful: Response Headers =", result_data.headers, "\n"
+        if VERBOSITY >= 4:
+            print "restful: Status code =", result_data.status_code, "\n"
+        return {'json':{}, 'text':result_data.text, 'status':result_data.status_code,
+                'headers':result_data.headers,
+                'timeout':False}
+    else:
+
+        if VERBOSITY >= 9:
+            print "restful: JSON = \n"
+            print json.dumps(result_data.json(), sort_keys=True, indent=4)
+        if VERBOSITY >= 6:
+            print "restful: Response Headers =", result_data.headers, "\n"
+        if VERBOSITY >= 4:
+            print "restful: Status code =", result_data.status_code, "\n"
+        return {'json':result_data.json(), 'text':result_data.text,
+                'status':result_data.status_code,
+                'headers':result_data.headers,
+                'timeout':False}
+
+def remote_shell(shell_cmd, timeout=300, user=OBM_CONFIG['ora']['username'] , password=OBM_CONFIG['ora']['password']):
+    '''
+    This function is used to run ipmitool on ova
+    This function is port form FIT Test, courtesy of George Paulos.
+    '''
+    logfile_redirect = None
+    address = arg_ora
+    if VERBOSITY >= 4:
+        print "remote_shell: Host =", address
+        print "remote_shell: Command =", shell_cmd
+
+    if VERBOSITY >= 9:
+        print "remote_shell: STDOUT =\n"
+        logfile_redirect = sys.stdout
+
+    # if localhost just run the command local
+    if address == 'localhost':
+        (command_output, exitstatus) = \
+            pexpect.run("sudo bash -c \"" + shell_cmd + "\"",
+                        withexitstatus=1,
+                        events={"assword": password + "\n"},

```

Comment:
This is intended. Because some times the shell password prompt is "Password" and sometime  it is "password"